docs/settings/infrastructure-ui-settings.asciidoc

```diff
@@ -0,0 +1,39 @@
+[role="xpack"]
+[[infrastructure-ui-settings-kb]]
+=== Infrastructure UI Settings in Kibana
+++++
+<titleabbrev>Infrastructure UI Settings</titleabbrev>
+++++
+
+You do not need to configure any settings to use Infrastructure UI. It is enabled by default.
+
+[float]
+[[general-infra-ui-settings-kb]]
+==== General Infrastructure UI Settings
+
+xpack.infra.enabled:: Set to `false` to disabled the Logs and Infrastructure UI plugin {kib}. Defaults to
```

Comment:
Make "disabled" present tense and add the missing word "in":

`to disable the Logs and Infrastructure UI plugin in {kib}`